### PR TITLE
Issue 81

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

File: tests/test_authorization.py [hunk 1]
```diff
@@ -363,3 +363,68 @@ def test_osa_versions(dispatcher_long_living_fixture, dispatcher_test_conf, role
 
     if expected_error is not None:        
         assert jdata["error"] == expected_error
+
+
+@pytest.mark.isgri_plugin
+@pytest.mark.isgri_plugin_dummy
+# some combinations of data rights and IC might be disfavored, but let's not enforce it now
+@pytest.mark.parametrize("roles", ["", "integral-private-qla"])
```
Comment (burnout87) [inline tests/test_authorization.py:371]: don't we test the `integral-public-nrt` role?

**Review by burnout87 — COMMENTED** [review]
which role do we want to test here?

File: tests/test_authorization.py (hunk 1 above)
Comment (volodymyrss) [reply tests/test_authorization.py:371]: this role should never be assigned or allowed

Comment (volodymyrss) [reply tests/test_authorization.py:371]: the desired behavior is to deny. there should be a better way to achieve this than with role

Comment (burnout87) [reply tests/test_authorization.py:371]: the role is not a bad approach, it will enable us to change this behavior in the future and start allowing NRT

Comment (volodymyrss) [reply tests/test_authorization.py:371]: > 

It's just this is almost for sure not going to happen. 

there should be a place for plugins to exclude impossible parameter combinations. is there such place? if not, we should add it:

https://github.com/oda-hub/dispatcher-app/issues/297

Actually, this was applicable in https://github.com/oda-hub/dispatcher-plugin-integral/blob/master/tests/test_authorization.py#L286 , similarly postponed

Comment (volodymyrss) [reply tests/test_authorization.py:371]: I will add the role test for now with with a note to exclude it

**Review by burnout87 — APPROVED** [review]

**Pull request merged**